libs/utils/fedsHelpers.js

```diff
@@ -0,0 +1,30 @@
+import { getConfig } from './utils.js';
```

Comment:
@bandana147 - would it be possible to rename the file to `federated` or something else that's more appropriate? Thanks!

Reply:
Done